ui-v2/app/templates/dc/services/show/services.hbs

```diff
@@ -0,0 +1,33 @@
+<div id="services" class="tab-section">
+  <div role="tabpanel">
+  {{#if (gt gateway.Services.length 0)}}
+    <section>
+      <p>
+        The following services may recieve traffic from external services through this gateway. Learn more about configuring gateways in our 
+        <a href={{env 'CONSUL_DOCS_URL'}}>step-by-step guide.</a>
```

Comment:
Does this need a specific URL?

Reply:
Actually, try and source this from someone too before you merge, otherwise this might get forgotten. If we can't source anything now, then if you can make a note to come back to it , that'd be great.

Reply:
I've added the new links